Source/WebCore/html/HTMLMediaElement.cpp

```diff
@@ -2343,18 +2350,20 @@ static inline bool isAllowedToLoadMediaURL(HTMLMediaElement& element, const URL&
     return element.document().contentSecurityPolicy()->allowMediaFromSource(url);
 }
 
-bool HTMLMediaElement::isSafeToLoadURL(const URL& url, InvalidURLAction actionIfInvalid)
+bool HTMLMediaElement::isSafeToLoadURL(const URL& url, InvalidURLAction actionIfInvalid, bool shouldLog) const
 {
     if (!url.isValid()) {
-        ERROR_LOG(LOGIDENTIFIER, url, " is invalid");
+        if (shouldLog)
+            ERROR_LOG(LOGIDENTIFIER, url, " is invalid");
```

Comment:
Nit: this could be simplified to `ERROR_LOG_IF(shouldLog, LOGIDENTIFIER, url, " is invalid")`